konf/raw-site.yaml

```diff
@@ -0,0 +1,57 @@
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: ubuntu-com-security-api
+spec:
+  selector:
+    app: ubuntu-com-security-api
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: ubuntu-com-security-api
+spec:
+  replicas: 5
+  template:
+    metadata:
+      labels:
+        app: ubuntu-com-security-api
+    spec:
+      containers:
+        - name: ubuntu-com-security-api
+          image: prod-comms.docker-registry.canonical.com/ubuntu-com-security-api:${TAG_TO_DEPLOY}
+          ports:
+            - name: http
+              containerPort: 80
+          envFrom:
+            - configMapRef:
+                name: global-envvars
+                optional: true
+            - configMapRef:
+                name: proxy-config
+                optional: true
+          env:
+            - name: DATABASE_URL
+              secretKeyRef:
```

Comment:
Not the valid definition.

Reply:
Fixed by using `konf production konf/site.yaml`.